tests/boards/ehl_crb/gpio/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.8.2)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
```

Comment:
For `project( )`, please use a brief, descriptive name, such as examples: `gpio_counter`, or `fade_led`